src/code-analysis/comment-collector.js

```diff
@@ -129,7 +129,8 @@ function applyMemberRoles(members, data) {
       _member.modifiers.push('static');
     }
 
-    if (member.definition.includes('function')) {
+    // (\(.*\)\s*=>)|\b\s*\(\s*\)|(function)
+    if (/(\(.*\)\s*\s*{)|(\(.*\)\s*\s*=>)/.test(member.definition)) {
```

Comment:
Why \s* defined 2x?

Why not leave function as one option and simplify to
/function|\(.*\)\s*=>/

Reply:
I was trying to target the parens of a function but I think this is probably too repetitive.

Reply:
Can you think of any reason not to use this regex? 
```
/function|=>/
```
Originally I thought I needed to capture the arguments and parens when I was working on this but that's not actually the case. I think this can probably suffice.

Reply:
That makes it very readable.  I can not think of a way it would give false positive for what it is intended for.